hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -394,6 +394,33 @@
         }
       ],
     },
+    //=========================================================================
+    // Flash & Passthrough CSRs
+    { name: "INTERCEPT_EN"
+      desc: '''Intercept Passthrough datapath.
+
+        '''
+      swaccess: "rw"
+      hwaccess: "hro"
+      fields: [
+        { bits: "0"
+          name: "status"
+          desc: "If set, Read Status is processed internally."
+        } // f: status
+        { bits: "1"
+          name: "jedec"
+          desc: "If set, Read JEDEC ID is processed internally."
+        } // f: jedec
+        { bits: "2"
+          name: "sfdp"
+          desc: "If set, Read SFDP is processed internally."
+        } // f: sfdp
+        { bits: "3"
+          name: "mbx"
```

Comment:
ah i guess this means we can still choose to pass the mailbox address through right?

Reply:
Right.

I assume we won't intercept entire read commands. Just return internally for the mailbox address.